components/admin_console/workspace-optimization/dashboard.tsx

```diff
@@ -141,32 +151,38 @@ const WorkspaceOptimizationDashboard = (props: Props) => {
         setTimeout(() => testSiteURL(onSuccess, onError, location.origin), 1000);
     }, []);
 
-    const dataKey = Object.keys(data);
+    const accData: AccordionDataType[] = [];
+    const dataKeys = Object.keys(data);
+
+    dataKeys.forEach((key: string) => {
+        const items = data[key].items.map((item) => (
+            <AccordionItem
+                key={`${key}-item_${item.id}`}
+                iconColor={getItemColor(item.status)}
+            >
+                <h5><i className={classNames('icon', {'icon-check-circle-outline': item.status === 'none', 'icon-alert-outline': item.status === 'warning', 'icon-alert-circle-outline': item.status === 'error'})}/>{item.title}</h5>
+                <p>{item.description}</p>
+            </AccordionItem>
+        ));
+        accData.push({
+            title: data[key].title,
+            description: data[key].description,
+            icon: data[key].icon,
```

Comment:
```suggestion
        const {title, description, icon} = data[key];
        accData.push({
            title,
            description,
            icon,
```

Reply:
with the beforementioned usage of `Object.entries()` you would not even need to destruct from a member access but just from the data object that is being passed in from the `entries` method